src/shared.zig

```diff
@@ -96,7 +96,13 @@ pub fn App(comptime Context: type) type {
 
                         // std.log.info("file(\"{}\", \"{}\")", .{ std.zig.fmtEscapes(src), std.zig.fmtEscapes(dst) });
 
-                        var file = try std.fs.cwd().openFile(src, .{});
+                        var file = std.fs.cwd().openFile(src, .{}) catch |err| switch (err) {
+                            error.FileNotFound => {
+                                std.log.err("file not found: {s}", .{src});
+                                return err;
+                            },
+                            else => return err,
+                        };
```

Comment:
this was just for trying to trace an error i had, not necessary or worth forwarding